en/docs/learn/configuring-data-sharing-for-business-nominated-representatives-configuration.md

```diff
@@ -0,0 +1,77 @@
+# Enabling the Business User Profile
+
+There are 2 profile types as `individual` and `organization`. Following are the 2 ways to get the profile type and you can configure.
```

Comment:
We have to mention that there can be multiple organization profiles according to the consumer's accounts

Reply:
let's fix this in next PR. @imesh94